.sparse/minimal-sdk

```diff
@@ -2,134 +2,97 @@
 /.sparse/minimal-sdk
 
 # Git
-/mingw64/bin/git.exe
-/mingw64/libexec/git-core/
-/mingw64/bin/libiconv-*[0-9].dll
-/mingw64/bin/libintl-*[0-9].dll
-/mingw64/bin/libpcre2-[1-9]*-*[0-9].dll
-/mingw64/bin/libssp-*[0-9].dll
-/mingw64/bin/zlib*[0-9].dll
+/clangarm64/bin/git.exe
+/clangarm64/libexec/git-core/
+/clangarm64/bin/libiconv-*[0-9].dll
+/clangarm64/bin/libintl-*[0-9].dll
+/clangarm64/bin/libpcre2-[1-9]*-*[0-9].dll
+/clangarm64/bin/zlib*[0-9].dll
 
 # system gitconfig
 /etc/gitconfig
 
-# GCC and friends
-/mingw64/bin/ar.exe
-/mingw64/bin/as.exe
-/mingw64/bin/cc.exe
-/mingw64/bin/gcc.exe
-/mingw64/bin/ld.exe
-/mingw64/bin/libgmp-*[0-9].dll
-/mingw64/bin/libgcc_s_seh-*[0-9].dll
-/mingw64/bin/libisl-*[0-9].dll
-/mingw64/bin/libwinpthread-*[0-9].dll
-/mingw64/bin/libzstd.dll
-/mingw64/bin/strip.exe
-/mingw64/bin/windres.exe
-/mingw64/lib/gcc/x86_64-w64-mingw32/*/cc1.exe
-/mingw64/lib/gcc/x86_64-w64-mingw32/*/collect2.exe
-/mingw64/lib/gcc/x86_64-w64-mingw32/*/*.o
-/mingw64/lib/gcc/x86_64-w64-mingw32/*/include/
-/mingw64/lib/gcc/x86_64-w64-mingw32/*/libgcc*
-/mingw64/lib/gcc/x86_64-w64-mingw32/*/liblto*
-/mingw64/lib/gcc/x86_64-w64-mingw32/*/libssp*
+# Clang and friends
+/clangarm64/bin/ar.exe
+/clangarm64/bin/as.exe
+/clangarm64/bin/cc.exe
+/clangarm64/bin/clang.exe
+/clangarm64/bin/ld.exe
+/clangarm64/bin/libgmp-*[0-9].dll
+/clangarm64/bin/libclang.dll
+/clangarm64/bin/libclang-cpp.dll
+/clangarm64/bin/libLLVM-*[0-9].dll
+/clangarm64/bin/libwinpthread-*[0-9].dll
+/clangarm64/bin/libzstd.dll
+/clangarm64/bin/llvm-strip.exe
+/clangarm64/bin/windres.exe
+/clangarm64/lib/clang/*/include/
+/clangarm64/lib/clang/*/lib/
```

Comment:
Is all of this really needed in the minimal SDK? `libLLVM-15.dll` alone is ~96MB already...

Reply:
I fear that's the price of using a low-level virtual machine ;-)